src/tests/Microsoft.Diagnostics.NETCore.Client/EventPipeSessionTests.cs

```diff
@@ -210,15 +210,20 @@ public async Task StartEventPipeSessionWithoutStackwalkTestAsync(TestConfigurati
                         // This exception can happen if the target process exits while EventPipeEventSource is in the middle of reading from the pipe.
                         runner.WriteLine($"Error encountered while processing events {ex}");
                     }
-                    finally
-                    {
-                        runner.WakeupTracee();
-                    }
                 });
                 runner.WriteLine("Waiting for stream Task");
-                streamTask.Wait(10000);
+                try
+                {
+                    await streamTask.WaitAsync(TimeSpan.FromMilliseconds(10000));
+                }
+                catch (TimeoutException)
+                {
+                    // Continue even if the stream task doesn't complete within the timeout
+                    runner.WriteLine("Stream task did not complete within timeout, proceeding to stop session");
+                }
                 runner.WriteLine("Done waiting for stream Task");
                 session.Stop();
```

Comment:
This test looked correct to me as it was before. session.EventStream.CopyTo isn't supposed to complete until the runtime closes the stream, which it shouldn't do until the session ends, which shouldn't happen until we call session.Stop().

If the runtime stops the trace prior to us calling Stop() that would be a runtime bug where the test is supposed to fail. If the process exits without us calling stop, that is either a runtime bug, some environmental issue randomly killing our test process, or a test issue that where we coded the tracee to exit too early.

I'd suggest we just re-enable the test without making any changes to the body of this method. If the issue re-occurs then we should gather more diagnostics (logging/dump) on why the process exits earlier than expected.